lib/menu.rb

```diff
@@ -0,0 +1,13 @@
+module Menu
+  def dishes_list
+    @dishes_list = { Burger: 5, Chips: 3, Drink: 1 }
+  end
+
+  def dishes_names
+    dishes_list.keys
+  end
+
+  def dishes_prices
+    dishes_list.values
+  end
+end
```

Comment:
Maybe the menu should be a class and it shouldn't be hardcoded. Everything has to be reusable :)